examples/analytics-iframe-transport-remote-frame.html

```diff
@@ -4,24 +4,26 @@
     <meta charset="UTF-8">
     <title>Requests Frame</title>
     <script>
-      window.addEventListener('amp-iframeTransportClientCreated', () => {
-        /**
-         * To receive AMP analytics events in a third-party frame, you must
-         * pass a callback function to this method. The callback will be called
-         * when an event is received, and will be passed two parameters:
-         * @param {string} event A string of the format specified in the
-         * requests block of the amp-analytics JSON config
-         * @param {string} transportId An ID uniquely identifying which creative
-         * generated the event
-         */
-        window.iframeTransportClient.onAnalyticsEvent(
-            (event, transportId) => {
-              // Now, do something meaningful with the AMP Analytics event
-              console.log('The page at: ' + window.location.href +
-                  ' has received an event: ' + event +
-                  ' from the creative with transport ID: ' + transportId);
-            });
-      });
+      /**
```

Comment:
nit: can we move the documentation to the correct place where `event => {}`, and fix the JSON doc .

Reply:
Done